src/classes/MyHandler/offer/accepted/updateListings.ts

```diff
@@ -151,7 +153,7 @@ export default function updateListings(
                         `(+ ${priceFromOptions.keys > 0 ? `${pluralize('key', priceFromOptions.keys, true)}, ` : ''}${
                             priceFromOptions.metal
                         } ref)` +
-                        `\nhttps://www.prices.tf/items/${sku}`;
+                        (isCustomPricer ? '' : `\nhttps://www.prices.tf/items/${sku}`);
```

Comment:
Maybe we can add something here instead of it being blank? Something to specify the price was fetched from the custom pricer.

Reply:
Oh sure thing, but I am not sure what's the best thing to put.

Reply:
I'm thinking something like "Price fetched from custom auto-pricer"

Reply:
Alright, will do that I guess.